src/components/LineChart/components/CustomTooltip/CustomTooltip.style.ts

```diff
@@ -0,0 +1,41 @@
+import { rem } from 'polished';
+import { Theme } from 'theme';
+
+export const tooltipStyle = () => (theme: Theme) => {
+  return {
+    fontSize: theme.typography.fontSizes['14'],
+    padding: rem(8),
+    color: theme.palette.white,
+    background: theme.utils.getColor('darkGray', 600),
+    opacity: '90%',
+    borderRadius: '4px',
+    minWidth: rem(200),
+    whiteSpace: 'nowrap' as const,
+  };
+};
+
+export const tooltipHrStyle = () => (theme: Theme) => {
+  return {
+    margin: '17px 0px 13px 0px',
+    height: '1px',
+    borderWidth: 0,
+    backgroundColor: theme.palette.white,
+    opacity: '10%',
+  };
+};
+
+export const tooltipUlStyle = () => () => {
+  return { padding: '0px', margin: '0px' };
+};
+
+export const tooltipLiStyle = () => (theme: Theme) => {
+  return {
+    listStyleType: 'none',
+    color: theme.palette.white,
+    width: '100%',
+    display: 'flex',
+    justifyContent: 'space-between',
+    height: '16px',
```

Comment:
instead of using px, try using the rem() function from polished and the standard spacing from ictinus (where possible)

eg: 16px -> rem(theme.spacing.md). This can apply for all pixel spacings from 4px to 32px with rounding when needed. For bigger spacings you can use only the rem function (eg. rem(200))

Reply:
thanks for the comment. done!